code/obj/artifacts/artifact_items/activator_key.dm

```diff
@@ -11,7 +11,7 @@
 	validtypes = list("ancient","martian","wizard","eldritch","precursor")
 	automatic_activation = 1
 	react_xray = list(12,80,95,8,"COMPLEX")
-	examine_hint = "It kinda looks like it's supposed to be inserted into something."
+	examine_hint = SPAN_ARTHINT("It kinda looks like it's supposed to be inserted into something.")
```

Comment:
The text changes should be where it is output, not here; this goes for all applications of this text macro.